student-preferred-name-portlet-api/src/main/java/edu/wisc/portlet/preferred/form/PreferredName.java

```diff
@@ -9,6 +9,8 @@
     
     private String middleName;
     
+    private String lastName;
```

Comment:
Probably a best practice to bump the `serialVersionUID` upon adding this member, but in practice it probably doesn't make any different for anything.